components/utils/highCharts.ts

```diff
@@ -10,13 +13,33 @@ export const plotLinesColor = "#FF0000"; // red;
 export const timeSeriesColors = originalHighchartsColors!.slice(1);
 
 // Colours from Bang Wong palette, see https://davidmathlogic.com/colorblind
-const colorBlindSafeColors = [
+export const colorBlindSafeColors = [
   { name: "Vermillion", hex: "#d55e00" },
   { name: "Bluish green", hex: "#009e73" },
   { name: "Sky blue", hex: "#56b4e9" },
 ];
 
-export const costsPieColors = ["rgba(1,1,1,0)"].concat(Array.from<string>(colorBlindSafeColors.map(color => color.hex))); // Make the center circle translucent
+// Create a range of N colors varying in brightness
+// NB This won't work well with colors that are already very light or very dark.
+export const getColorVariants = (colorHex: string, numberOfVariants: number) => {
```

Comment:
Is this mimicking what highcharts as doing for us anyway with the colour variants in the pie chart? It doesn't do something similar for stacked bars?

Reply:
Yeah, there's nothing equivalent for stacked bars. Highcharts provides `colorByPoint`, but this produces variation _within_ a series, which is an entity that is orthogonal to the stacked bar.